### PR TITLE
move to sonatype maven central

### PR DESCRIPTION
**Pull request by jaredsburrows**

### CODE REVIEW

**Comment by jaredsburrows** [comment]
Needs https://github.com/Kotlin/kotlinx.html/issues/173 to be fixed

File: build.gradle [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   id 'groovy'
+  id 'java-gradle-plugin'
   id 'maven-publish'
+  id 'signing'
 }
 
 repositories {
-  jcenter()
+  mavenCentral()
+  jcenter() // Remove when https://github.com/Kotlin/kotlinx.html/issues/173 is fixed
```
Comment (nisrulz) [inline build.gradle:15]: Seems like this is not a blocker anymore: https://github.com/Kotlin/kotlinx.html/issues/173#issuecomment-826522662

File: src/test/groovy/com/jaredsburrows/license/LicensePluginAndroidSpec.groovy [hunk 2]
```diff
@@ -46,7 +46,8 @@ final class LicensePluginAndroidSpec extends Specification {
       """
       buildscript {
         repositories {
-          jcenter()
+          mavenCentral()
+          jcenter() // Remove when https://github.com/Kotlin/kotlinx.html/issues/173 is fixed
```
Comment (nisrulz) [inline src/test/groovy/com/jaredsburrows/license/LicensePluginAndroidSpec.groovy:50]: Can be removed now? https://github.com/Kotlin/kotlinx.html/issues/173#issuecomment-826522662

**Comment by nisrulz** [comment]
Is there anything keeping this PR from being merged? I was able to notice something about kotlin-html, which seems to be fixed now.

Happy to help, if needed.

**Pull request merged**